content/blog/2021-02-01-tidynhl-analyse/index.fr.md

```diff
@@ -0,0 +1,710 @@
+```
+
+```
+##    draft_year draft_overall       player_name skater_points goalie_wins
+## 1:       2005            44      Paul Stastny           756           0
+## 2:       2007            14 Kevin Shattenkirk           387           0
+## 3:       2009            33     Ryan O'Reilly           572           0
+## 4:       2009             3      Matt Duchene           638           0
+## 5:       2009            64      Tyson Barrie           357           0
+## 6:       2011             2 Gabriel Landeskog           485           0
+## 7:       2013             1  Nathan MacKinnon           509           0
+## 8:       2015            10    Mikko Rantanen           260           0
+```
+
+Un peu moins de "bons choix" que les Blue Jackets, mais ceux-ci semblent avoir eu un impact bien plus grand. On peut dire que l'Avalanche a su profiter de leurs hauts choix au repêchage ...
+
+Finalement, aviez-vous réussi à deviner quels étaient les 5 "bons choix" de notre Sainte-Flanelle? 
+
+
+```r
+# Bons choix des Canadiens
+dt_good_picks[good_pick == TRUE & team_drafted == "MTL", (columns_show), with = F]
+```
+
+```
+##    draft_year draft_overall       player_name skater_points goalie_wins
+## 1:       2005             5       Carey Price             0         352
+## 2:       2005           200  Sergei Kostitsyn           176           0
+## 3:       2007            43       P.K. Subban           429           0
+## 4:       2007            22    Max Pacioretty           565           0
+## 5:       2010           147 Brendan Gallagher           341           0
+```
+
+Pas trop mal, mais sur 10 années de repêchages, on aurait bien aimé avoir quelques "bons choix" de plus (surtout en première ronde) ...
+
+<center>
+
+![](https://media.giphy.com/media/toYIetMaC2fdqfRifg/giphy.gif)
+
+</center>
+
+# Conclusion {#conclusion}
+
+Pour conclure cet article, je dois vous avouez que je continu de croire que le repêchage est une science inexacte. Il est difficile de tirer des conclusions évidentes étant donné les nombreuses composantes à prendre en compte. Sans entrer trop dans les détails, il y a le nombre de choix et les positions des joueurs repêchés. Certains joueurs font moins de points, mais apportent une composante de plus à une équipe, comme le leadership ou même l'aspect défensif. Toutefois, selon mes analyses, voici les équipes que je considère comme les grands "gagnants" et "perdants" ainsi que pourquoi.
+
+## Les grands "gagnants" 👍
+
+- **Avalanche du Colorodo:** On peut voir leurs "bon choix" plus haut. Ils ont bien saisis leur chance sur leurs choix de 1ère ronde, considérant qu'ils en ont eu que 9 alors que la moyenne de la ligue se situe à 11.
+- **Bruins de Boston**: Arrivant au 26ème rang pour le nombre de choix, ils sont pourtant au 6ème rang pour les points récoltés par les joueurs repêchés. Ils sont également dans le premier tier pour le nombre de "bons choix".
```

Comment:
tier -> tiers

Reply:
Non, c'est un mot français tiers @davebulaval ([source : Larousse](https://www.larousse.fr/dictionnaires/francais/tiers/78041))

Reply:
Dans le sens qu'on l'utilise non, je voir ça comme le premier tier comme étant en tête de peloton pas un tier dans le sens 1/3 ou un tier comme une troisième personne.

Reply:
Je suis pas mal sur que @stecaron insinuait que les Bruins étaient dans les 10 meilleures équipes, ou le premier tiers des équipes. Une manière simple de trancher serait de savoir comment tu aurais dit ta phrase en anglais :

1. They also are in the top third of teams when counting the number of "good choices" over the years.
2. They also are a first-tier team when counting the number of "good choices" over the years.

Pour moi, l'option 1 a plus de sens. Si on voulait avoir le sens de l'option 2, j'écrirais plutôt :

Ils sont également une équipe de premier rang pour le nombre de "bon choix".

Bref, si on veut insinuer qu'ils sont dans le top-10 en disant "premier tiers", il n'y a pas d'anglicisme. Si on ne veut pas nécessairement insinuer qu'ils sont dans le top-10, mais qu'ils sont performants, je reformulerais.

Reply:
Guys, je vais laisser tiers de même ... j'pense pas que [Simon Jolin-Barrette](https://www.oqlf.gouv.qc.ca/office/organigramme.pdf) va venir voir mon article et me tapper sur les doigts si jamais c'est pas ledgit ...